action.yml

```diff
@@ -33,11 +33,12 @@ runs:
         distribution: "zulu"
 
       # CodeSee Maps Go support uses a static binary so there's no setup step required.
-    - name: Configure Node.js 16
+      
+    - name: Configure Node.js 18
       uses: actions/setup-node@v3
-      if: ${{ fromJSON(steps.detect-languages.outputs.languages).javascript }}
+      if: ${{ fromJSON(steps.detect-languages.outputs.languages).javascript || fromJSON(steps.detect-languages.outputs.languages).kotlin }}
```

Comment:
question: I don't see kotlin as a supported language at the language-detection: https://github.com/Codesee-io/codesee/tree/main/packages/codesee/src/map/langs . Do we need to implement it first for it to work?

Reply:
Yes. It’s in a different PR

Reply:
We had to revert it because of this issue.